### DISCUSSION TITLE
Application for study package rejected

### DISCUSSION THREAD

**Discussion by Brok-23** [community/community#19487] [GitHub Education] [answered]
The application for the study package was rejected immediately. I don't think I have been effectively reviewed. I can provide all the information I need. Please explain the reasons for the immediate rejection

**Comment by melvincwng** [comment]
@Brok-23, can you elaborate more on this issue?

What was the reason of rejection? Was it due to unclear ID or insufficient documents to prove your status as a student?

**Reply by Brok-23** [reply]
I tried six times and used different methods of proof. All of them were rejected immediately after submission

**Comment by melvincwng** [comment, marked as answer]
@Brok-23, my suggestion would be to raise a support ticket to Github Education Support over [here](https://support.github.com/contact/education).

Describe the details of your issue, and provide several working days for the support engineers at GitHub to investigate and get back to you.

In the meantime, try to reapply again for the GitHub Student Developer Pack. Refer to this [Github Docs link](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-for-your-schoolwork/apply-for-a-student-developer-pack) for a detailed step-by-step guide.

**Reply by melvincwng** [reply]
@Brok-23 , if this reply was helpful, please 'Mark as answer'.

Thanks, and have a good day 😄.

**Reply by melvincwng** [reply]
You can refer to #19398 for more information as well.

**Reply by Brok-23** [reply]
thanks

**Comment by vamsi3856** [comment]
Did you upload a document with date for your verification?

**Reply by Brok-23** [reply]
can only taking photos, no upload option

**Reply by vamsi3856** [reply]
okay..! Try once again ..
